templates/node/node.py

```diff
@@ -170,12 +170,8 @@ def usable_storagepool(sp):
                 except ValueError:
                     pass  # no zdb filesystem on this storagepool
 
-            # all path used by installed zerodb services
```

Comment:
this part is useless, we only care about the paths so we can exclude them from `fs_paths`